Gradle.gitignore

```diff
@@ -1,14 +1,76 @@
-.gradle
-/build/
+*.class
+*.bat
+*.setting
 
-# Ignore Gradle GUI config
-gradle-app.setting
+libs/
 
-# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
-!gradle-wrapper.jar
+# Mobile Tools for Java (J2ME)
```

Comment:
And most of those are already part of `Java.gitignore` - that is typically used together with `Gradle.gitignore`.